README.md

```diff
@@ -6,7 +6,7 @@
 
 This installation method provides the full Home Assistant experience on a regular operating system. This means, all components from the Home Assistant method are used, except for the Home Assistant Operating System. This system will run the Home Assistant Supervisor. The Supervisor is not just an application, it is a full appliance that manages the whole system. It will clean up, repair or reset settings to default if they no longer match expected values.
 
-By not using the Home Assistant Operating System, the user is responsible for making sure that all required components are installed and maintained. Required components and their versions will change over time. Home Assistant Supervised is provided as-is as a foundation for community supported do-it-yourself solutions. We only accept bug reports for issues that have been reproduced on a freshly installed, fully updated Debian with no additional packages.
+By not using the [Home Assistant Operating System](https://github.com/home-assistant/operating-system), the user is responsible for making sure that all required components are installed and maintained. Required components and their versions will change over time. Home Assistant Supervised is provided as-is as a foundation for community supported do-it-yourself solutions. We only accept bug reports for issues that have been reproduced on a freshly installed, fully updated Debian with no additional packages.
```

Comment:
Lets's link this to our official documentation instead.

Reply:
Even better! I found pages that mentioned the HA OS on here - https://www.home-assistant.io/installation/ - but they seem to split out depending on the type of device you're "installing" to - and more commonly it seemed, just imaging. 

And if I go look at the [Releases](https://github.com/home-assistant/operating-system/releases/) page for example, I don't see a particular "generic"/separate installer, with the most general seeming to be a VHD or OVA. 

The [GitHub link](https://github.com/home-assistant/operating-system) above seems to give a fairly good overview/intro to HA OS, and if they want to get started, that section will send them to the [getting started guide](https://www.home-assistant.io/getting-started/) and [installation instructions](https://www.home-assistant.io/hassio/installation/) - though Installation Instructions sends to `/hassio/installation/` and redirects to `/installation/`

Is there a Particular page that outlines this better or give a better overview of the HA OS to use?

Reply:
I think that people need to use our documented installation instructions on the website. Just pointing them to the OS repository might be nice, but isn't a complete story.

From the installation pages in our documentation they, for example, might come to the conclusion that a Container-based installation is a better fit.